devices/tuya/_TZE200_pay2byax_door_sensor.json

```diff
@@ -0,0 +1,69 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": ["_TZE200_pay2byax", "_TZE200_kltffuzl", "_TZE200_fwoorn8y"],
+  "modelid": ["TS0601", "TS0601", "TS0601"],
+  "product": "ZG-102Z",
+  "sleeper": true,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_OPEN_CLOSE_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0xef00"
+      ],
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion"
```

Comment:
Does it make sense here to add the usual Tuya firmware magic?

Reply:
updated